cabal-install/src/Distribution/Client/CmdHaddockProject.hs

```diff
@@ -161,208 +161,216 @@ haddockProjectAction flags _extraArgs globalFlags = do
   -- we need.
   --
 
-  withContextAndSelectors RejectNoTargets Nothing
-                          (commandDefaultFlags CmdBuild.buildCommand)
-                          ["all"] globalFlags HaddockCommand
-                          $ \targetCtx ctx targetSelectors -> do
-    baseCtx <- case targetCtx of
-      ProjectContext -> return ctx
```

Comment:
I suspect this all is fourmolu and it is *not* good. What has been happening here, @Kleidukos?